packages/arb-token-bridge-ui/src/hooks/useArbTokenBridge.ts

```diff
@@ -208,18 +213,24 @@ export const useArbTokenBridge = (
       return error.message
     }
 
-    addTransaction({
-      type: 'deposit-l1',
-      status: 'pending',
-      value: utils.formatUnits(amount, nativeCurrency.decimals),
-      txID: tx.hash,
-      assetName: nativeCurrency.symbol,
-      assetType: AssetType.ETH,
+    addPendingTransaction({
       sender: walletAddress,
-      // TODO: change to destinationAddress ?? walletAddress when enabling ETH transfers to a custom address
```

Comment:
might as well keep the comment

Reply:
It's quite obvious change for ETH transfers and we have an issue for that. I don't think the comment is needed